docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -814,3 +814,49 @@ files, and a `.tar.gz` of the raw DICOM data as acquired during the candidate
 scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
 
 Only `GET` is currently supported.
+
+### 5.3 Upload DICOM file
+```
+PUT /candidates/$CandID/$VisitLabel/dicoms/$Filename/$IsPhantom
+```
+
+Upload a DICOM file that is compressed as a `.zip`,`.tar.gz` or `.tgz`.
+$CandID must be the full name of the file and its extension.
+$IsPhantom must be `'Y'` or `'y'` if it's a phantom file, else `'N'` or `'n'` 
+if not.
+
+Returns JSON data upon success having the form:
+
+```js
+{
+    "success":"Uploaded",
+    "mri_upload_id":"3"
+}
+```
+
+mri_upload_id is the Upload ID for the file.
```

Comment:
This should also describe where the data goes after it's `PUT` to the server. Does it end up in the dicom archive? Or the imaging browser? Somewhere else?

Reply:
Yes indeed. I will add that. 

There are supposed to be two steps (based on what i thought was the existing Loris way of doing thngs)

1. Upload the file. It will be automatically deposited/saved in the location specified in Configuration->Paths->MRI-Upload Directory.

This will also insert the file details into the mri_upload table and return the Upload ID in the HTTP response.

If "ImagingUploader Auto Launch" is enabled, then the Imaging pipeline is automatically triggered. Otherwise nothing else will happen.

2. Trigger the Imaging pipeline (explicitly) by giving the Upload Id as well as the other details to process the files.